srcpkgs/libqb/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'libqb'
+pkgname=libqb
+version=1.0.3
+revision=1
+hostmakedepends="glib-devel pkg-config"
+short_desc="libqb is a library with the primary purpose of providing high-performance, reusable features for client-server architecture, such as logging, tracing, inter-process communication (IPC), and polling."
+only_for_archs="x86_64"
+build_style=gnu-configure
```

Comment:
the order of the variables is wrong, please use `xlint` from the `xtools` packages to get the correct ordering.